backend/src/main/java/com/rota/exceptions/StaffNotFoundException.java

```diff
@@ -1,6 +1,8 @@
 package com.rota.exceptions;
 
 public class StaffNotFoundException extends RuntimeException {
+  static final long serialVersionUID = 1L;
```

Comment:
What does this do?

Reply:
I was getting a warning that a Serializable class needs a serialVersionUID, so this helps to suppress that.

Some explanation here
https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it#285809

Reply:
Ah right didn't realise `RuntimeException` implemented `Serializable`!

Out of interest where did you see the warning?

Reply:
Neovim with coc.nvim plugin complains.
Given that this plugin is basically vscode's intellisense, I think vscode will complain too.